src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -105,12 +98,30 @@ public static void setDashboards(String user, Set<String> projects, Set<String>
 		else {
 			List<String> common = new ArrayList<String>(indexPatterns);
 			
+			// build normalizedSort
+			List<String> normalizedSort = new ArrayList<String>();
```

Comment:
Actually, I wrote my suggestion incorrectly.  It should have been:

List<String> normalizedSort = new ArrayList<>();

Its just a shortcut where you need to type less.